airflow_tecton/operators/job_utils.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Tecton, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import pprint
+import time
+
+
+def wait_until_completion(hook, workspace, feature_view, job_id):
```

Comment:
maybe wait_until_success? It seems like it throws if it fails right

Reply:
i think wait_until_completion is technically fine though, but maybe worth adding a brief pydoc explaining this